README.md

```diff
@@ -1,70 +1,166 @@
-# Getting Started with Create React App
 
```

Comment:
- [ ] This change is being highlighted here because you do not have any changes in the `src/components/category.js` file. So I am unable to highlight the code. Kindly add some code to the category component and it should have the "Check status" button